facebook-commerce-pixel-event.php

```diff
@@ -285,15 +285,16 @@ public function get_event_script( $event_name, $params, $method = 'track' ) {
 		 * @param string $method     Name of the pixel's fbq() function to call.
 		 */
 		public function inject_event( $event_name, $params, $method = 'track' ) {
-
+			// If we have add to cart redirect enabled, we must defer the events to render them the next page load.
+			$defer = 'yes' === get_option( 'woocommerce_cart_redirect_after_add', 'no' );
 			if ( \WC_Facebookcommerce_Utils::isWoocommerceIntegration() ) {
-				\WC_Facebookcommerce_Utils::wc_enqueue_js( $this->get_event_code( $event_name, self::build_params( $params, $event_name ), $method ) );
+				\WC_Facebookcommerce_Utils::wc_enqueue_js( $this->get_event_code( $event_name, self::build_params( $params, $event_name ), $method ), $defer );
 			} else {
+				// @TODO: if this one is ever used. Doubts have I.
```

Comment:
That is not a comment for the production code. Either please verify if this is used or remove the comment. Or in the worst case scenario, which is that we still don't know please create an issue to explore this further.